metadata-ingestion/src/datahub/ingestion/source/usage/starburst_trino_usage.py

```diff
@@ -211,10 +218,16 @@ def _get_joined_access_event(self, events):
             )
 
             if not event_dict.get("usr"):
+                self.report.num_joined_access_events_skipped += 1
                 logging.info("The username parameter is missing. Skipping ....")
                 continue
 
-            joined_access_events.append(TrinoJoinedAccessEvent(**event_dict))
+            try:
+                joined_access_events.append(TrinoJoinedAccessEvent(**event_dict))
+            except Exception as e:
+                self.report.num_joined_access_events_skipped += 1
+                logger.error("Error while parsing TrinoJoinedAccessEvent.", e)
```

Comment:
logger.something doesn't accept multiple arguments - it differs from print in that way

```suggestion
                logger.info(f"Error while parsing TrinoJoinedAccessEvent: {e}")
```